components/clamav/clamav-milter.p5m

```diff
@@ -0,0 +1,83 @@
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+# Copyright (c) 2016 Jim Klimov. All rights reserved.
+# Partially inspired by SFE manifest
+#   http://sfe.opencsw.org/localhostoi151a8/manifest/0/antivirus%2Fclamav@0.98,5.11-0.0.151.1.8%3A20131210T210011Z
+#
+
+# pull the manpages out of the component dir
+<transform file path=usr/share/man/(man3/.+$) -> set action.hash %<\1> >
+
+<transform file path=usr.*/man/.+ -> default mangler.man.stability uncommitted>
+set name=pkg.fmri \
+    value=pkg:/antivirus/$(COMPONENT_NAME)-milter@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY) for e-mail relays"
+set name=pkg.description value="$(COMPONENT_SUMMARY) for e-mail relays"
+set name=info.classification value="org.opensolaris.category.2008:Applications/System Utilities"
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license COPYING license="GPL2"
+
+depend fmri=antivirus/clamav-core type=require
+### The above line should ensure common ClamAV dependencies du-jour
+### as well as "clamav" UID and GID
+
+#depend fmri=library/pcre type=require
+### Required by core since 0.98.x
+#depend fmri=library/security/openssl type=require
+#group groupname=clamav
+#user ftpuser=false gcos-field="ClamAV Reserved UID" group=clamav password=NP username=clamav
+
+### Sendmail AND its libmilter are needed for clamav-milter.
+#depend fmri=service/network/smtp/sendmail type=require
+
+file \
+    files/var/svc/manifest/site/clamav-milter.xml \
+    path=var/svc/manifest/site/clamav-milter.xml \
+    group=sys mode=0444 owner=root \
+    restart_fmri=svc:/system/manifest-import:default
+
+file \
+    etc/clamav-milter.conf.sample \
+    path=etc/clamav-milter.conf.sample-v$(COMPONENT_VERSION) \
+    group=sys mode=0444 owner=root
+file \
+    files/etc/clamav-milter.conf.sol \
+    path=etc/clamav-milter.conf.sol \
+    group=sys mode=0444 owner=root
+
+file \
+    files/var/clamav/log/clamav-milter.log \
+    path=/var/clamav/log/clamav-milter.log \
+    mode=0644 owner=clamav group=clamav preserve=custom
+
+link path=var/log/clamav-milter.log target=../clamav/log/clamav-milter.log
```

Comment:
fixed (removed the empty logs and symlinks; made a /var/log/clamav/ symlink to directory instead)